rom/laghos_rom.cpp

```diff
@@ -165,6 +165,10 @@ void printSnapshotTime(std::vector<double> const &tSnap, std::string const path,
 {
     cout << var << " snapshot size: " << tSnap.size() << endl;
     std::ofstream outfile_tSnap(path + var);
+
+    typedef std::numeric_limits< double > dbl;
```

Comment:
Instead of the `typedef` could there just be `std::numeric_limits<double>::max_digits10`?

Reply:
Yes it can and should be changed. There is a point when the code does not work and I change from `max_digits10` to this `typedef`, but I later figure out the `max_digits10` is not the source of the problem.